server/lib/nl/config_builder/base.py

```diff
@@ -75,6 +75,53 @@ def _replace_special(input_string_title_case: str) -> str:
   return input
 
 
+def _count_caps(word: str) -> int:
```

Comment:
Perhaps its time to move these functions to a title_utils.py ?

Reply:
Will do. Deferred this particular functionality to the next PR. Removed from here.